src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.tsx

```diff
@@ -62,10 +85,25 @@ const CreateOrEditNote = ({
       sx={styles.container}
     >
       <Box sx={styles.header}>
-        <Avatar />
-        <Typography variant={TypographyVariantEnum.Subtitle2}>
-          {userNameMocked}
-        </Typography>
+        {loading ? (
```

Comment:
Move this complex render logic before `return` statement to avoid polluting our jsx code

Reply:
Fixed